File: Python/백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
a = int(input())
for i in range(a):
    aa = list(map(int,input().split()))
    n = aa.pop(0)
    avg = sum(aa)/n
    cnt = 0
    for j in aa:
        if j>avg:
            cnt+=1
    print(f"{(cnt/n)*100:.3f}%")

File: C#/백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0;i<a;i++){
                double[] aa = Array.ConvertAll(Console.ReadLine().Split(),double.Parse);
                double n = aa[0];
                double avg = (aa.Sum()-n)/n;
                double cnt = 0;
                for (int j=1;j<aa.Length;j++){
                    if (aa[j]>avg){
                        cnt++;
                    }
                }
                Console.WriteLine($"{(cnt/n*100):N3}%");
            }
        }
    }
}